src/sentry/api/permissions.py

```diff
@@ -55,6 +55,12 @@ def has_object_permission(self, request, view, obj):
     def is_project_key(self, request):
         return isinstance(request.auth, ProjectKey)
 
+    def get_scopes(self, key):
+        try:
+            return self.scope_map[key]
```

Comment:
> probably should just use .get(), and not entirely sure the function call is all that useful

You're right. It's useless now. Removed, and switched everything to `scope_map.get(request.method, [])` instead.